src/Api/Parser/AbstractRestParser.php

```diff
@@ -93,6 +93,12 @@ private function extractHeader(
     ) {
         $value = $response->getHeaderLine($shape['locationName'] ?: $name);
 
+        if (!empty($shape->offsetGet('jsonvalue'))
```

Comment:
I added this additional check to make sure that this would fail if for some reason the variable was set to empty string.